nativelib/src/main/scala/scala/scalanative/native/unistd.scala

```diff
@@ -0,0 +1,8 @@
+package scala.scalanative
+package native
+
+@extern
+object unistd {
+  def sleep(seconds: CUnsignedInt): Int = extern
+  def usleep(usecs: Long): Int          = extern
+}
```

Comment:
Given that this one is posix API and might not work on windows, lets put those in `scala.scalanative.runtime` package. We shouldn't exposes it as part of our public API due to portability constraints.